### ISSUE TITLE
Composer with PHP 8.0.1 not satisfying requirements

### ISSUE THREAD

**Issue by robertbakker** [pascaldevink/shortuuid#77]
Hoi Pascal, 

I'm having a problem updating my dependencies with Composer 2.
```
Problem 2
    - pascaldevink/shortuuid is locked to version 3.0 and an update of this package was not requested.
    - pascaldevink/shortuuid 3.0 requires php ^7.3|8.0 -> your php version (8.0.1) does not satisfy that requirement.
```
It seems 8.0 is too strict and doesn't take point releases like 8.0.1 into account.

Groet

**Comment by jdreesen** [comment]
That's right. It should probably be `^8.0` instead.

**Comment by pascaldevink** [comment]
Fixed in 3.0.1
